client/app/scripts/components/node-details/node-details-health-link-item.js

```diff
@@ -23,7 +24,7 @@ export function appendTime(url, time) {
     const json = decodeURIComponent(url.substr(pos + cloudLinkPathEnd.length));
     try {
       payload = JSON.parse(json);
-      payload.time = { queryEnd: time.unix() };
+      payload.time = { queryEnd: moment(time).unix() };
```

Comment:
tests need adjustment for `time` parameter

Reply:
Good call, by fixing the tests, I realized `appendTime` function wasn't handling all the cases right so I fixed it as well :)